src/Miew.js

```diff
@@ -1878,6 +1878,8 @@ Miew.prototype.load = function (source, opts) {
     }
   }
 
+  viewInterpolator._reset();
```

Comment:
The underscore means a private method. Let's not extend our technical debt at least in the new commits :)

Reply:
Done